paper_framework_sources/p2079_system_execution_context.md

```diff
@@ -80,9 +91,7 @@ public:
   system_context& operator=(const system_context&) = delete;
   system_context& operator=(system_context&&) = delete;
 
-  system_scheduler get_scheduler();
-  implementation-defined_delegation_sender execute_chunk() noexcept;
-  implementation-defined_delegation_sender execute_all() noexcept;
+  implementation-defined-system_scheduler get_scheduler();
   size_t max_concurrency() noexcept;
```

Comment:
Should we make ``size_t max_concurrency() noexcept`` ->``size_t max_concurrency() const noexcept``?

Reply:
Yes.